mw_organisation_activity.go

```diff
@@ -59,29 +59,54 @@ func (k *OrganizationMonitor) ProcessRequestLive(r *http.Request) (error, int) {
 		return errors.New("this organisation access has been disabled, please contact your API administrator"), 403
 	}
 
-	// We found a session, apply the quota limiter
-	reason := k.sessionlimiter.ForwardMessage(&session,
+	// We found a session, apply the quota and rate limiter
+	reason := k.sessionlimiter.ForwardMessage(
+		&session,
 		k.Spec.OrgID,
-		k.Spec.OrgSessionManager.Store(), false, false)
```

Comment:
it was set to `false, false` so `k.sessionlimiter.ForwardMessage` was doing basically nothing when it is not in `ExperimentalProcessOrgOffThread` mode